classes/ActionScheduler_Store.php

```diff
@@ -124,5 +124,11 @@ public static function instance() {
 		}
 		return self::$store;
 	}
+
+	/**
+	 * @param string $claim_id
+	 *
+	 * @return array
+	 */
+	abstract public function find_actions_by_claim_id( $claim_id );
```

Comment:
Just one thing to call out here: I added this to ensure IDE code completion works properly. I didn't see this class extended anywhere else. If this is a class that *is* often extended elsewhere, I can change this into a noop method instead of an abstract method to prevent errors for anyone updating.

Reply:
@JPry we actually had a patch which was adding this already in an outstanding PR - #87 (see [this commit specifically](https://github.com/Prospress/action-scheduler/pull/87/commits/2b0cebd3e6bc47baf1919baed60b007f71c1a8b4)).

Now that #87 has been merged, could you please:

1. remove the commit which introduced this
1. rebase the rest of the PRs commits on top of the `1-6-dev` branch
1. change the PR to be against the `1-6-dev` branch instead of master (we'll release this with 1.6)

> In regards to Travis failing, it looks like it's unrelated to any changes I made... Is there something that needs to be done about that?

That should also be fixed once rebasing on top of `1-6-dev` as it was fixed in #87 with https://github.com/Prospress/action-scheduler/pull/87/commits/4802f134c53815d37370d9e6edd9f5052b36f320 :)